app/views/home.php

```diff
@@ -1595,186 +1595,21 @@
-                                    </div>
-                                </a>
-                            </div>
-                            <div class="item-5 hover-up wow animate__animated animate__fadeIn">
-                                <a href="/companies/show">
-                                    <div class="item-logo">
-                                        <div class="image-left">
-                                            <img alt="jobBox" src="assets/imgs/brands/brand-2.png"></div>
-                                        <div class="text-info-right">
-                                            <h4>Square</h4><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><span class="font-xs color-text-mutted ml-10"><span>(</span><span>16</span><span>)</span></span>
-                                        </div>
-                                        <div class="text-info-bottom mt-5"><span class="font-xs color-text-mutted icon-location">New York, US</span><span class="font-xs color-text-mutted float-end mt-5">37<span> Open Jobs</span></span></div>
-                                    </div>
-                                </a>
-                            </div>
-                            <div class="item-5 hover-up wow animate__animated animate__fadeIn">
-                                <a href="/companies/show">
-                                    <div class="item-logo">
-                                        <div class="image-left">
-                                            <img alt="jobBox" src="assets/imgs/brands/brand-8.png"></div>
-                                        <div class="text-info-right">
-                                            <h4>Vista</h4><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><img alt="jobBox" src="assets/imgs/template/icons/star.svg"><span class="font-xs color-text-mutted ml-10"><span>(</span><span>97</span><span>)</span></span>
+                            <?php foreach($data['companies'] as $company): ?>
+                                <div class="item-5 hover-up wow animate__animated animate__fadeIn">
+                                    <a href="/companies/show">
+                                        <div class="item-logo">
+                                            <div class="image-left">
+                                                <img alt="jobBox" src="<?php echo $company['logo']; ?>">
```

Comment:
Sử dụng short syntax `<?= ?>`
```suggestion
                                                <img alt="sửa jobBox này lại thành $companies['name']" src="<?= $company['logo']; ?>">
```